### PR TITLE
gh-117655: Fix `test_strptime` raises a DeprecationWarning

### PR DESCRIPTION
**Pull request by nineteendo**
* Issue: gh-117655

### CODE REVIEW

**Comment by nineteendo** [comment]
@pganssle, could you review this?

**Comment by nineteendo** [comment]
@erlend-aasland, does this look better?

**Comment by nineteendo** [comment]
@AlexWaygood, is this something you could review?

**Comment by erlend-aasland** [comment]
> @erlend-aasland, does this look better?

I'm not the correct person to review this. I'll request Greg's review, since he authored the recent change that introduced this deprecation warning.

**Comment by nineteendo** [comment]
OK.

File: Lib/test/datetimetester.py [hunk 1]
```diff
@@ -2793,6 +2794,7 @@ def test_strptime_single_digit(self):
                 newdate = strptime(string, format)
                 self.assertEqual(newdate, target, msg=reason)
 
+    @warnings_helper.ignore_warnings(category=DeprecationWarning)  # gh-70647
```
Comment (Eclips4) [inline Lib/test/datetimetester.py]: I think we can omit this comment since just a two lines below we have the same explanation :)

**Review by Eclips4 — APPROVED** [review]

File: Lib/test/datetimetester.py (hunk 1 above)
Comment (nineteendo) [reply Lib/test/datetimetester.py]:
```suggestion
    @warnings_helper.ignore_warnings(category=DeprecationWarning)
```

**Comment by nineteendo** [comment]
Thanks, should I ask on Discourse for someone to review this as well?

**Review by gpshead — APPROVED** [review]

**Comment by nineteendo** [comment]
Thanks, I didn't dare pinging you after the comment of Erlend.

**Comment by gpshead** [comment]
no problem, i'm always overflowing with notifications so I easily miss things or have other priorities at times.  thanks for the cleanup!  I probably should've done this type of thing when making the original change. thanks for the pings!

**Pull request merged**